samples/boards/stm32/power_mgmt/suspend_to_ram/src/main.c

```diff
@@ -131,18 +222,22 @@ int main(void)
 		printk("error: random device not ready");
 	}
 
+	spi_loopback_setup();
+
 	printk("Device ready\n");
 
 	while (true) {
 		gpio_pin_configure_dt(&led, GPIO_OUTPUT_ACTIVE);
 		adc_test();
+		test_spi_loopback();
```

Comment:
For now spi test should be conditional to the fact that S2RAM is not enabled

Reply:
Done